examples/erc20/AlphaWallet-Discovery-Token/discover.en.shtml

```diff
@@ -0,0 +1,74 @@
+<script type="text/javascript"><![CDATA[
+class Token {
+
+    constructor(tokenInstance) {
+        this.props = tokenInstance;
+        this.getStatus();
```

Comment:
Because getStatus() is async, if the user loads the action view and press the sign button quick enough and if the network or server is slow, the `claimed` check might not have returned. You'll have to judge if it's ok for experimenting since there's animation involved to show the screen, as well as time for the user to see the confirm prompt and sign (but of course the anyone can just look at the URL and hit it multiple times with curl).